### PR TITLE
Add documentation for sending emails with Symfony on GAE

### PR DESCRIPTION
**Pull request by Sethbass**
Hi @bshaffer,

As discussed in the issue #1021 , I have enriched the documentation about the emails with Symfony and GAE. 

Cheers,

Sebastien

### CODE REVIEW

**Comment by Sethbass** [comment]
@googlebot I fixed it.

File: appengine/php72/symfony-framework/README.md [hunk 1]
```diff
@@ -265,12 +265,76 @@ the [Stackdriver Error Reporting UI][stackdriver-errorreporting-ui]! If you copi
 `https://YOUR_PROJECT_ID.appspot.com/en/logging/notice` and
 `https://YOUR_PROJECT_ID.appspot.com/en/logging/exception`
 
+## Send emails
+
+As stated in the [Migration guide from PHP 5.x to 7.x][cloud-migration] the Mail API or App Engine has been removed.
```
Comment (bshaffer) [inline appengine/php72/symfony-framework/README.md]: This first sentence here can be removed, as I imagine most people coming to this tutorial will not have previously run services on App Engine for PHP 5.5.

File: appengine/php72/symfony-framework/README.md [hunk 2]
```diff
@@ -265,12 +265,76 @@ the [Stackdriver Error Reporting UI][stackdriver-errorreporting-ui]! If you copi
...
+
+### Install
+
+First you need to install the mailer component:
+
+```
+composer require symfony/mailer
+```
+
+Then depending on the provider you chose, you need to add the additional component, [Symfony mailer documentation][symfony-mailer]
```
Comment (bshaffer) [inline appengine/php72/symfony-framework/README.md]: Change this to:

```
For this example, we will use `Mailgun`.  To use a different mail provider, see the [Symfony mailer documentation][symfony-mailer].
```

File: appengine/php72/symfony-framework/README.md [hunk 3]
```diff
@@ -265,12 +265,76 @@ the [Stackdriver Error Reporting UI][stackdriver-errorreporting-ui]! If you copi
...
+You can now send emails in Controller and Service as usual:
+```
+// src/Controller/MailerController.php
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\Email;
+
+class MailerController extends AbstractController
```
Comment (bshaffer) [inline appengine/php72/symfony-framework/README.md]: Change this to `ExampleController`, to make it clear that creating a `MailerController` is not required to use the mailer

**Comment by Sethbass** [comment]
@bshaffer , I have done the suggested changes in my last commit. as soon as this one is merged I will open a new one for the upload tutorial. 

Thanks,

Sebastien

**Comment by bshaffer** [comment]
@Sethbass This looks great! Unfortunately, the google bot thinks you haven't signed the CLA. Your last 3 commits have the user `Administer` applied to them, with no profile image. Any idea why? I believe this is causing the issue.

Can you try squashing your changes to a single commit and force-pushing to your branch?

**Comment by bshaffer** [comment]
@Sethbass I resolved the issues! If you can comment `@googlebot I consent` on #1030 wee will be all set!

**Comment by Sethbass** [comment]
@googlebot I consent

**Comment by bshaffer** [comment]
Submitted in #1030!! Thanks again @sethbass!

**Pull request closed without merging**